main/main.cpp

```diff
@@ -513,6 +514,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --build-solutions                 Build the scripting solutions (e.g. for C# projects). Implies --editor and requires a valid project to edit.\n");
 	OS::get_singleton()->print("  --dump-gdextension-interface      Generate GDExtension header file 'gdextension_interface.h' in the current folder. This file is the base file required to implement a GDExtension.\n");
 	OS::get_singleton()->print("  --dump-extension-api              Generate JSON dump of the Godot API for GDExtension bindings named 'extension_api.json' in the current folder.\n");
+	OS::get_singleton()->print("  --dump-extension-api-with-docs    Generate JSON dump of the Godot API (including documentation) for GDExtension bindings named 'extension_api.json' in the current folder.\n");
```

Comment:
This feels a little too repetitive: when skimming quickly it's hard to spot the difference between this one and the one above. Many of the existing option descriptions make reference to other options - maybe this option could refer to the one above? For example, "Generate JSON dump like the previous option, but including documentation."